schema/manifest_backwards_compatibility_test.go

```diff
@@ -110,14 +110,14 @@ func TestBackwardsCompatibilityManifestList(t *testing.T) {
 			fail: false,
 		},
 	} {
-		sum := sha256.Sum256([]byte(tt.manifest))
+		sum := sha256.Sum256([]byte(tt.manifestlist))
```

Comment:
While we're in here, let's start using the digest package to perform these kinds of operations.